.github/scripts/torchao_model_releases/eval.sh

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
Do we have these copyright in all files in ao?

Reply:
yeah, this is enforced by some lint I think